src/components/dataset/list.vue

```diff
@@ -11,9 +11,9 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <div>
-    <dataset-introduction/>
     <dataset-table/>
     <loading :state="datasets.initiallyLoading"/>
+    <dataset-introduction/>
```

Comment:
Is there any space between `DatasetIntroduction` and the loading message above it? You can check this by adding [network throttling](https://css-tricks.com/throttling-the-network/) in Chrome devtools. (I use custom network throttling profiles with various latency numbers.) If not, how about adding some `margin-top` to `DatasetIntroduction`?

Reply:
Actually, looking at the screenshot in the release criteria, there seems to be a bit of space between `DatasetTable` and `DatasetIntroduction`. So that might be another reason to add some `margin-top` to `DatasetIntroduction`.

Reply:
Our tables have `margin-bottom: 20px`. And loading message with the image look fine to me:

![image](https://user-images.githubusercontent.com/447837/213505597-30964c5f-a71c-4627-bbbc-0aa608012d20.png)

I can hide the image during loading though

Reply:
Either way, I think it's OK to show the image during loading.